docs/modules/ROOT/pages/tutorials/getting-started/java-web.adoc

```diff
@@ -0,0 +1,53 @@
+= Getting Started on {product} with the OpenShift Web Console using Java
+
+// THIS FILE IS AUTOGENERATED
+// DO NOT EDIT MANUALLY
+
+image:logos/java.svg[role="related thumb right",alt="Programming language logo",width=120,height=120] This tutorial explains how to run applications written in the Java programming language on {product} using the OpenShift web console.
```

Comment:
This Java logo overlaps with the hr below. All other programming language icons are OK.

Reply:
Fixed